<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color:  rgb(85, 77, 72);
border-radius:15px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:10px;
background-color: rgb(170, 150, 158);
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0.0454545, y1:0.545, x2:1, y2:0, stop:0 rgba(230, 43, 89, 255), stop:1 rgba(255, 255, 255, 255));
}

QLabel{
color: rgb(255,255,254);
font:12pt&quot;SansSerif&quot;;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Consultar  | Base de Datos</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>370</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_minimize">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imgs/-.png</normaloff>imgs/-.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_normal">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imgs/+.png</normaloff>imgs/+.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imgs/x.png</normaloff>imgs/x.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_contenido">
         <property name="styleSheet">
          <string notr="true">QPushButton{
font:12pt &quot;SansSerif&quot;;
color: rgb(255,255,254);
border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.0454545, y1:0.545, x2:1, y2:0, stop:0 rgba(230, 43, 89, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.017, x2:1, y2:0.0227273, stop:0 rgba(81, 40, 31, 255), stop:0.988636 rgba(54, 0, 54, 255));
}

QLineEdit{
background-color: rgb(22,22,26);
border-radius:10px;
font: 12pt &quot;SansSerif&quot;;
color: rgb(255,255,255);
border: 2px solid rgb(114,117,126);
}

QLineEdit{
background-color: rgb(22,22,26);
border-radius:10px;
font: 12pt &quot;SansSerif&quot;;
color: rgb(255,255,255);
border: 2px solid rgb(44,182,125);
}

QLineEdit:focus{
background-color: rgb(1,1,1);
border-radius:10px;
font: 12pt &quot;SansSerif&quot;;
border: 2px solid rgb(127,90,240);
}
QLabel{
color: rgb(255,255,254);
font:12pt&quot;SansSerif&quot;;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
          <item>
           <widget class="QFrame" name="frame_datos">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,1,1,1,1">
             <property name="leftMargin">
              <number>40</number>
             </property>
             <property name="topMargin">
              <number>40</number>
             </property>
             <property name="rightMargin">
              <number>40</number>
             </property>
             <property name="bottomMargin">
              <number>40</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Registrar en la Base de Datos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="in_nombre">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>Nombre</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="in_correo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>Correo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="in_telefono">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>Telefono</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_importar_img">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string> Subir Imagen</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>imgs/up.png</normaloff>imgs/up.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="img_preview">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
               <item>
                <widget class="QPushButton" name="btn_limpiar">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Limpiar</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_guardar">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Guardar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_buscar">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1,3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>40</number>
             </property>
             <property name="topMargin">
              <number>40</number>
             </property>
             <property name="rightMargin">
              <number>40</number>
             </property>
             <property name="bottomMargin">
              <number>40</number>
             </property>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Consultar Datos</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
               <item>
                <widget class="QLineEdit" name="in_buscar_nombre">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="placeholderText">
                  <string>Nombre</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_buscar">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>imgs/search.png</normaloff>imgs/search.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="correo">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Telefono:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="telefono">
               <property name="text">
                <string>Correo:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="imagen">
               <property name="text">
                <string>Foto</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
